spec/features/trusted_login_flow_spec.rb

```diff
@@ -19,15 +19,38 @@
     { sp: OpenStax::Api::Params.sign(params: payload, secret: @app.secret) }
   }
 
+  describe 'arriving with an existing account' do
+
+    it 'pre-fills email on sign in when there is a match' do
+      user = create_user 'user'
+      create_email_address_for(user, payload[:email])
+      arrive_from_app(params: signed_params)
+      expect(page).to have_field('login_username_or_email', with: payload[:email])
+      click_button(t :"signup.start.next")
+      complete_login_password_screen 'password'
+      expect_back_at_app
+      expect(user.external_uuids.where(uuid: payload[:uuid])).to exist
+    end
+
+    it 'auto signs in and returns when linked' do
```

Comment:
Is this spec a dupe of `describe 'coming from app when already linked'` below?

Reply:
yes it is. I'll remove it and move the terms spec up